src/SIL.XForge/Services/EmailService.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using MailKit;
+using MailKit.Net.Smtp;
+using MimeKit;
+
+namespace SIL.XForge.Services
+{
+    public class EmailService : IEmailService
+    {
+        public string SendEmail(string emailId, string subject, string body, string domain, string name, string smtpServer, string portNumber)
```

Comment:
I would inject `SiteOptions` into this class instead of passing `smtpServer` and `portNumber` into this method.

Reply:
We injected SiteOptions to EmailServer's constructor.